Project-Hotpot/Project-Hotpot/ViewControllers/HomeViewController.swift

```diff
@@ -46,7 +45,7 @@ class HomeViewController: UIViewController {
     }
     
     @IBAction func panCard(_ sender: UIPanGestureRecognizer) {
-        let card = sender.view!
+        guard let card = sender.view else { return }
```

Comment:
curious if this line works without the { } around return. Typically we always format brackets in newline

Reply:
whenever I try removing the braces, the compiler screams at me, saying  "Expected '{' after 'guard' else}  "

Reply:
Ignore my comment, looks like it is a standard patter in swift from looking at our codebase. I see people use { return} in one line if it is short and going to newline if there is longer code